static/src/javascripts/projects/journalism/views/audioSeriesContributions.html

```diff
@@ -1,11 +1,16 @@
 <section class="flagship-audio__contributions-banner fc-container">
     <div class="fc-container__inner">
         <div class="fc-container__header"> </div>
-        <div class="fc-container--rolled-up-hide fc-container__body">
-            <h2>Support The Guardian</h2>
-            <p>Help us deliver the independent
-                <br> journalism the world needs</p>
-            <a class="contributions__contribute contributions__option-button" data-link-name="audio-series-contributions">Support the Guardian</a>
+        <div class="fc-container--rolled-up-hide fc-container__body flagship-audio__contributions-banner-holder">
+            <div class="flagship-audio__contributions-banner-text">
+                <h2>Support The Guardian</h2>
+                <p>Help us deliver the independent
+                    <br> journalism the world needs</p>
+                <a class="contributions__contribute contributions__option-button" data-link-name="audio-series-contributions">Support the Guardian</a>
```

Comment:
no `href`?

Reply:
eek! good catch. Must have removed this at some point